cookiecutter.json

```diff
@@ -1,5 +1,5 @@
 {
-    "project_name": "pybamm-example-project",
+    "project_name": "pybamm_example_project",
```

Comment:
I'm curious, do hyphens in the `project_name` not work? I ask because we do convert the hyphens to underscores in `__project_slug` below.

Reply:
Yes I was getting confused between two naming conventions in a lot of different files and modules, `pybamm-example-project` cannot be a module name while `pybamm_example_project` can. So I just made it standard for everything to stick to one naming convention to avoid confusions in multiple files.

Reply:
> pybamm-example-project cannot be a module name while pybamm_example_project can

I think this is a bug, in this case. The name of the project in `pyproject.toml` (and therefore on PyPI) can very well be `pybamm-example-project`, but the files for such a project with hyphens must be placed in `src/pybamm_example_project`, so we are probably not converting the hyphens to underscores (i.e., missing the usage of the the slug variable perhaps).

Reply:
Oops sorry I didn't notice the slug variable was already configured, thanks! I will fix it.